app/scripts/views/ready.js

```diff
@@ -57,6 +59,29 @@ function (Cocktail, BaseView, Template, Session, Xss, Strings,
       return this._createMarketingSnippet();
     },
 
+    /**
+     * Creates the service name for the ready template
+     * Can either be a string or a link to the relier redirectUri
+     *
+     * @returns {String} serviceName
+     * @private
+     */
+    _createServiceName: function () {
+      var relier = this.relier;
+      var serviceName = relier.get('serviceName');
+      var redirectUri = relier.get('redirectUri');
+      var alwaysRedirect = relier.get('alwaysRedirect');
+
+      // if this relier wants to always redirect the user then create a link to the service
+      if (alwaysRedirect && redirectUri && Url.isNavigable(redirectUri)) {
```

Comment:
`alwaysRedirect` implies that the user would be redirected automatically, is that not the case?